### PR TITLE
Toast

### PR DESCRIPTION
**Pull request by Varu98**

### CODE REVIEW

File: styles/toast.css [hunk 1]
```diff
@@ -0,0 +1,30 @@
+@import url(./main.css);
+
+.bar-wrapper {
+  margin: 1rem;
+  display: flex;
+  background-color: var(--russian-violet);
+  max-width: 30rem;
+  color: #fff;
+  padding: 1rem 1rem;
```
Comment (BelideNikhil) [inline styles/toast.css:9]: this can just be 1rem

File: styles/toast.css [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  justify-content: flex-start;
+  text-align: center;
+  border-radius: 10px;
+  box-shadow: 0 5px 5px rgba(0, 0, 0, 0.37);
```
Comment (BelideNikhil) [inline styles/toast.css:13]: use this as a variable

File: pages/toast.html [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    <link rel="stylesheet" href="../styles/main.css" />
+    <title>SnackBar Component</title>
+</head>
+
+<body>
+    <div class="wrapper">
+        <div class="bar-wrapper">
+            <div class="content">Can't send photo. Retry in 5 seconds</div>
+            <button class="action-btn">RETRY</button>
+            <button class="close-btn">X</button>
```
Comment (BelideNikhil) [inline pages/toast.html:17]: content name seems very generic, add something like toast-content.
Also use X icon instead of the letter itself.

**Review by BelideNikhil — APPROVED** [review]
Rest look good!

**Review by manojasarsa — APPROVED** [review]
LGTM 👍

File: pages/toast.html (hunk 3 above)
Comment (Varu98) [reply pages/toast.html:17]: Right!! Thanks for the feedback Nikhil !

**Pull request merged**